README.md

```diff
@@ -221,6 +223,23 @@ Further Information
 | elasticsearch_clusterinfo_last_retrieval_success_ts                   | gauge     | 1           | Timestamp of the last successful cluster info retrieval
 | elasticsearch_clusterinfo_up                                          | gauge     | 1           | Up metric for the cluster info collector
 | elasticsearch_clusterinfo_version_info                                | gauge     | 6           | Constant metric with ES version information as labels
+| elasticsearch_slm_stats_up                                            | gauge     | 0           | Up metric for SLM collector
+| elasticsearch_slm_stats_total_scrapes                                 | gauge     | 0           | Number of scrapes for SLM collector
+| elasticsearch_slm_stats_json_parse_failures                           | gauge     | 0           | JSON parse failures for SLM collector
+| elasticsearch_slm_stats_retention_runs                                | gauge     | 0           | Total retention runs
+| elasticsearch_slm_stats_retention_failed                              | gauge     | 0           | Total failed retention runs
+| elasticsearch_slm_stats_retention_timed_out                           | gauge     | 0           | Total retention run timeouts
+| elasticsearch_slm_stats_retention_retention_deletion_time_millis      | gauge     | 0           | Retention run deletion time
```

Comment:
This has retention twice, is that intentional? Also time metrics should be converted to seconds.